config_utilities_ros/config_utilities_ros/gui/README.md

```diff
@@ -0,0 +1,37 @@
+# Config Utilities Dynamic Config GUI
+
+# Installation
+
+## Dependencies
+Local:
+```bash
+pip install Flask webbrowser
```

Comment:
You only should need flask (I'm pretty sure that you're using the standard library webbrowser module)